frontends/web/src/components/accordion/accordion.module.css

```diff
@@ -0,0 +1,45 @@
+.accordion {
+  width: 100%;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  padding: 0;
+  background-color: var(--background-secondary);
+  border: none;
+  cursor: pointer;
+  font-size: 16px;
```

Comment:
nit: I think we should stick to `font-size: var(--size-default)`, so that all fonts in the dialog have same size.

Also that would help me when we move to responsive font sizes)